docs/reference/cluster/health.asciidoc

```diff
@@ -1,18 +1,44 @@
 [[cluster-health]]
 === Cluster Health
 
-The cluster health API allows to get a very simple status on the health
-of the cluster. For example, on a quiet single node cluster with a single index
-with one shard and one replica, this:
+[[cluster-health-api-request]]
+==== {api-request-title}
+
+`GET _cluster/health` +
+
+`GET /_cluster/health/test1,test2`
+
+[[cluster-health-api-desc]]
+==== {api-description-title}
+
+The cluster health API allows to get a simple status on the health of the 
+cluster. The API can also be executed against one or more indices to get just 
+the specified indices health.
+
+The cluster health status is: `green`, `yellow` or `red`. On the shard level, a 
+`red` status indicates that the specific shard is not allocated in the cluster, 
+`yellow` means that the primary shard is allocated but replicas are not, and 
+`green` means that all shards are allocated. The index level status is 
+controlled by the worst shard status. The cluster status is controlled by the 
+worst index status.
+
+One of the main benefits of the API is the ability to wait until the cluster 
+reaches a certain high water-mark health level. For example, the following will 
+wait for 50 seconds for the cluster to reach the `yellow` level (if it reaches 
+the `green` or `yellow` status before 50 seconds elapse, it will return at that 
+point):
 
 [source,js]
 --------------------------------------------------
-GET _cluster/health
+GET /_cluster/health?wait_for_status=yellow&timeout=50s
 --------------------------------------------------
 // CONSOLE
-// TEST[s/^/PUT test1\n/]
 
-Returns this:
+[[cluster-health-api-response-body]]
+==== {api-response-body-title}
+
+The API returns the following response in case of a quiet single node cluster 
```

Comment:
I believe the response body title should be parameter definitions.

You can move the example snippet to the `[[cluster-health-api-example]]` section.

Reply:
I added the param definitions to the section and moved the snippet to the examples. Please have a look at the param definitions because these were not provided. I created them, but some might be inexact.

Reply:
I'm not sure whether we should list the params in alphabetical order or rather in the same order as they are presented in the response body. Please let me know what do you prefer.

Reply:
Feel free to leave this section out for now.

If you decide to include it, let's go with the same order as the response body. Thanks @szabosteve!